packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.tsx

```diff
@@ -80,6 +82,30 @@ const LockedFunctionSettings = (props: Props) => {
         ]);
     }, [domain]);
 
+    function getPrepopulatedAriaLabel() {
+        let str = `Function with equation ${equationPrefix}${equation}`;
+
+        // Add the domain/range constraints to the aria label
+        // if they are not the default values.
+        if (domain && !(domain[0] === -Infinity && domain[1] === Infinity)) {
+            str += `, ${domainRangeText} from ${domain[0]} to ${domain[1]}`;
```

Comment:
I think, rather than using `${domainRangeText}`, we always want this word to be `domain`. IIUC, the domain of a function is always the set of input values, so if the function has the form `x = f(y)`, the domain is a set of y coordinates.

Might be good to confirm this with math content creators, though.

Reply:
That is a good point! I'll ask

Reply:
I'll change it to `domain` here and in the editor UI. If a change is requested in the future, we can updated it. For now, I'd like to unblock this so I can keep landing all the PRs stacked on top of this.